libsolidity/formal/BMC.h

```diff
@@ -204,6 +208,28 @@ class BMC: public SMTEncoder
 
 	/// Number of verification conditions that could not be proved.
 	size_t m_unprovedAmt = 0;
-};
 
+	enum class LoopControlKind
+	{
+		Continue,
+		Break
+	};
+
+	// Current path conditions and SSA indices for break or continue statement
+	struct LoopControl {
+		explicit LoopControl(
+			LoopControlKind _kind,
+			smtutil::Expression _pathConditions,
+			VariableIndices _variableIndicies
+		);
+
+
+		LoopControlKind kind;
+		smtutil::Expression pathConditions;
+		VariableIndices variableIndicies;
```

Comment:
```suggestion
		VariableIndices variableIndices;
```
I think this needs to be fixed similarly in other places too.